### PR TITLE
NEW : TAKEPOS-Clear search results

### PR DESCRIPTION
**Pull request by altairis-melina**

### CODE REVIEW

**Comment by eldy** [comment]
Can you provide more info on the goal of PR ?

**Comment by altairis-tof** [comment]
this PR is done to clear search results after a sale is done.

File: htdocs/takepos/index.php [hunk 1]
```diff
@@ -211,6 +211,7 @@ function ClearSearch() {
 	<?php if ($conf->browser->layout == 'classic') { ?>
 	setFocusOnSearchField();
 	<?php } ?>
+	if (typeof clearSearchResults != "undefined") $("#search").trigger('keyup');
```
[the remark this answers is not in the document]
Comment (altairis-tof) [reply htdocs/takepos/index.php:214]: @eldy you are tight, i will ask @altairis-melina to fix that. Thanks !

[the remark this answers is not in the document]
Comment (altairis-tof) [reply htdocs/takepos/index.php:214]: @altairis-melina please also replace the typeof test to boolean test as adviced by eldy

File: htdocs/takepos/invoice.php [hunk 2]
```diff
@@ -1014,7 +1014,7 @@ function fail($message)
 // Set focus to search field
 if ($action == "search" || $action == "valid") {
 	?>
-	parent.ClearSearch(true);
+	parent.ClearSearch(false);
```
Comment (altairis-tof) [inline htdocs/takepos/invoice.php:1017]: @altairis-melina why did you change ?

Comment (altairis-tof) [reply htdocs/takepos/invoice.php:1017]: @altairis-melina in fact, eldy asked you to add false parameter in the call of other ClearSearch, nut this one is ok
